backend/src/main/resources/application.yml

```diff
@@ -1,24 +0,0 @@
-# include common properties for every profile in this section
-
-spring:
-  datasource:
-    url: jdbc:postgresql://localhost:5432/planit
-    username: postgres
-    password: postgres
-  jpa:
-    hibernate:
-      jdbc:
-        lob:
-          non_contextual_creation: true
-      dialect: org.hibernate.dialect.PostgreSQLDialect
-      ddl-auto: update
-integrations:
-  google-token-redirect-uri: http://localhost:3000
-
----
-# profile specific properties
-
-spring:
```

Comment:
Why delete this file?

Reply:
same as before: generally this file contains project-based configuration and it should not be public